roboflow/__init__.py

```diff
@@ -8,7 +8,7 @@
 from roboflow.core.project import Project
 from roboflow.core.workspace import Workspace
 
-__version__ = "0.2.23"
+__version__ = "0.2.24"
```

Comment:
why the version was changed? Are you planning on a new release? It's better to make a single separate PR with the version bump to avoid overlapping with other PRs

Reply:
Noted! Appreciate the feedback! 

Is this still good to push or do I need to revert the version number?

Reply:
Now you have to push it since we increase the number from another PR :)